modelscope/hub/snapshot_download.py

```diff
@@ -185,6 +193,7 @@ def _snapshot_download(
     local_dir: Optional[str] = None,
     allow_patterns: Optional[Union[List[str], str]] = None,
     ignore_patterns: Optional[Union[List[str], str]] = None,
+    max_workers: int = 8,
```

Comment:
same setting as hf?

Reply:
Yes, the default value for `max_workers` in huggingface `snapshot_download` is 8.